src/Take.Elephant.Tests/ExpirableKeyMapFacts.cs

```diff
@@ -109,8 +110,8 @@ public async Task UpdateKeyExpirationDateSucceeds()
             var actual = await map.GetValueOrDefaultAsync(key);
             AssertEquals(actual, value);
             await Task.Delay(ttl + ttl + ttl);
```

Comment:
Keep tests assertions consistent: use same checks made on previous test

Reply:
Nice :)

For the `newTtl` variable though, I think it only makes sense in this test. This one is testing updatig/altering the TTL on an existing key. Thus why I felt the need to call it newTtl.

Not that I'm a fan of these `ttl + ttl + ttl` sprinkled around, but they're for another reason (make sure the delay is long enough for the test to work)